### ISSUE TITLE
Neural Network Information

### ISSUE THREAD

**Issue by haloxhanner** [luoyetx/Joint-Face-Detection-and-Alignment#6]
Please explain a bit about the Neural Network that you used in this project. 

How many hidden layers does it have? Have you performed data augmentation, batch normalization, ensembling or dropout? Could you explain them?

**Comment by luoyetx** [comment]
network definitions is under `proto` folder, basically follow the network from [paper](http://arxiv.org/abs/1604.02878).
